paasta_tools/cli/cmds/spark_run.py

```diff
@@ -441,6 +441,8 @@ def get_spark_config(
         "spark.executorEnv.PAASTA_INSTANCE": paasta_instance,
         "spark.executorEnv.PAASTA_CLUSTER": args.cluster,
         "spark.executorEnv.PAASTA_INSTANCE_TYPE": "spark",
+        "spark.executorEnv.AWS_SECRET_ACCESS_KEY": secret_key,
```

Comment:
non_user_args will failed if people configure this.  I don't think we want to fail people's job if they want to use a different creds.  Would you mind add another check and only set the value if it was not set?

Reply:
Good catch. Updated.